### PR TITLE
test: Add multiple tests

### PR DESCRIPTION
**Pull request by josemiguelmelo**

### CODE REVIEW

File: src/main/resources/docs/multiple-tests/use-patterns-specified/src/.tailor.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+format: xcode
+
+only:
+    - constant-naming
```
Comment (lolgab) [inline src/main/resources/docs/multiple-tests/use-patterns-specified/src/.tailor.yml]: Is this indenting correct?

File: src/main/resources/docs/multiple-tests/with-config/src/.tailor.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+format: xcode
+
+only:
+    - constant-naming
```
Comment (lolgab) [inline src/main/resources/docs/multiple-tests/with-config/src/.tailor.yml]: Same here?

File: src/main/resources/docs/multiple-tests/use-patterns-specified/src/.tailor.yml (hunk 1 above)
Comment (lolgab) [reply src/main/resources/docs/multiple-tests/use-patterns-specified/src/.tailor.yml]: Maybe you can 2 space indent

**Review by lolgab — CHANGES_REQUESTED** [review]

File: src/main/resources/docs/multiple-tests/with-config/src/.tailor.yml (hunk 2 above)
Comment (josemiguelmelo) [reply src/main/resources/docs/multiple-tests/with-config/src/.tailor.yml]: Done ✅

File: src/main/resources/docs/multiple-tests/use-patterns-specified/src/.tailor.yml (hunk 1 above)
Comment (josemiguelmelo) [reply src/main/resources/docs/multiple-tests/use-patterns-specified/src/.tailor.yml]: Done ✅

**Review by lolgab — APPROVED** [review]

**Pull request merged**